rust/src/lib.rs

```diff
@@ -1,12 +1,6 @@
-#![deny(clippy::all, clippy::perf, clippy::correctness)]
+#![deny(clippy::all)]
```

Comment:
when did this change? when I added these `clippy::all` did not contain these

Reply:
No idea, it's been like this in "my forever".